apps/frontline-widgets/src/messengerBundle.js

```
@@ -166,4 +166,4 @@
       background-image: ${f};
       background-size: ${A?"32px":"18px"};
       background-position: center;
-    `}});window.addEventListener("message",J);window.addEventListener("message",e=>p(void 0,null,function*(){var i;let{data:t}=e,{isVisible:n,message:c,isSmallContainer:m}=t||{};if(t.fromErxes&&t.source==="fromMessenger"){if(c==="playSound"){U();return}if(c==="unreadCount"){j((i=t.count)!=null?i:0);return}let s=o==null?void 0:o.querySelector(".erxes-launcher");if(!s)return console.error("Messenger: launcher element is not defined");y&&document.body.classList.toggle("widget-mobile",n),c==="messenger"&&(y&&n?R():O(),n?(l.classList.add("erxes-messenger-shown"),l.classList.remove("erxes-messenger-hidden"),s.style.backgroundImage="none",s.innerHTML=H,E(0)):(l.classList.remove("erxes-messenger-shown","erxes-messenger-expand"),l.classList.add("erxes-messenger-hidden"),s.style.backgroundImage=f,s.style.backgroundSize=A?"32px":"18px",s.innerHTML="",E(W))),b.classList.toggle("small",m)}}));})();
+    `}});window.addEventListener("message",J);window.addEventListener("message",e=>p(void 0,null,function*(){var d;let{data:t}=e,{isVisible:n,message:i,isSmallContainer:l}=t||{};if(t.fromErxes&&t.source==="fromMessenger"){if(i==="playSound"){U();return}if(i==="unreadCount"){j((d=t.count)!=null?d:0);return}let s=o==null?void 0:o.querySelector(".erxes-launcher");if(!s)return console.error("Messenger: launcher element is not defined");y&&document.body.classList.toggle("widget-mobile",n),i==="expandMessenger"&&(a.classList.remove("erxes-messenger-shown"),a.classList.add("erxes-messenger-expand")),i==="collapseMessenger"&&(a.classList.remove("erxes-messenger-expand"),a.classList.add("erxes-messenger-shown")),i==="messenger"&&(y&&n?R():O(),n?(a.classList.add("erxes-messenger-shown"),a.classList.remove("erxes-messenger-hidden"),s.style.backgroundImage="none",s.innerHTML=H,E(0)):(a.classList.remove("erxes-messenger-shown","erxes-messenger-expand"),a.classList.add("erxes-messenger-hidden"),s.style.backgroundImage=f,s.style.backgroundSize=A?"32px":"18px",s.innerHTML="",E(W))),b.classList.toggle("small",l)}}));})();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(`@typescript-eslint/no-unused-expressions`)

---

[error] 't' is never reassigned. Use 'const' instead.

(prefer-const)